Purchasing.Mvc/Controllers/OrderController.cs

```diff
@@ -894,6 +894,12 @@ public ActionResult Cancel(int id, string comment)
                 return RedirectToAction("Review", new {id});
             }
 
+            if (order.StatusCode.IsComplete)
```

Comment:
@jSylvestre isn't the purchaser supposed to be able to still cancel a completed order?  If not we'll need to update the docs